server/services/GameLogic.js

```diff
@@ -157,14 +176,31 @@ module.exports = class GameLogic {
                 socket.emitStatus(this.roomId, this.status);
                 socket.emitAllVotes(this.roomId, this.votes);
                 this.calcScores();
-                setTimeout(() => this.nextRound(), 5000);
+                this.nextRoundTimeout = setTimeout(() => this.nextRound(), 5000);
```

Comment:
You could probably move this 5000 into a variable too as it's used in multiple places